src/backends/odbc/standard-use-type.cpp

```diff
@@ -212,14 +203,19 @@ void odbc_standard_use_type_backend::pre_use(indicator const *ind)
     SQLSMALLINT sqlType(0);
     SQLSMALLINT cType(0);
     SQLLEN size(0);
+    SQLLEN bufLen(0);
 
     void* const sqlData = prepare_for_bind(size, sqlType, cType);
-
+    if (size > ODBC_MAX_COL_SIZE)
+    {
+        bufLen = size;
+        size = 0;
```

Comment:
I think it would be more clear if the comment about this 0 actually being `SQL_SS_LENGTH_UNLIMITED` were kept.